services/update_user_report.py

```diff
@@ -0,0 +1,28 @@
+import webapp2
+import json
+import logging
+
+from models.connexus_user import ConnexusUser
+
+class UpdateUserReport(webapp2.RequestHandler):
+
+    def post(self):
+
+        json_string = self.request.body
+        dict_object = json.loads(json_string)
+
+        user_id = dict_object['user_id']
+        report_rate = int( dict_object['report_rate'] )
+        
+        user_result = ConnexusUser.query(ConnexusUser.user_id == user_id).fetch()
+
+        for user in user_result:
+            user.report = report_rate
+            user.put()
+        
+        
+        res = { "msg" : "user updated", "success": True }
+
+        self.response.out.write(json.dumps(res))
```

Comment:
No need to write a response I think since it is a post method.

Reply:
The status code should be used for denoting a successful operation

Reply:
Sure, I'm used to handling detailed error messaging as responses for any type of request (regardless of type) for frontend display reasons, but I can update this..